src/transforms/groupby.js

```diff
@@ -39,11 +39,11 @@ exports.attributes = {
         valType: 'string',
         description: [
             'Pattern by which grouped traces are named. If only one trace is present,',
-            'defaults to the group name (`"%g"`), otherwise defaults to the group name',
-            'with trace name (`"%g (%t)"`). Available escape sequences are `%g`, which',
-            'inserts the group name, and `%t`, which inserts the trace name. If grouping',
+            'defaults to the group name (`"%{group}"`), otherwise defaults to the group name',
+            'with trace name (`"%{group} (%{trace})"`). Available escape sequences are `%{group}`, which',
```

Comment:
One small change though, right? Perhaps templateString should accept *multiple* objects from which to pull values and it takes the first one with a value. Then it `group` and `name` would be the first preference before grabbing `trace` properties. Specifically:

Now:
```javascript
Lib.templateString('%{group} (%{trace})', {group: ..., trace: ...})
```

Proposed:
```javascript
Lib.templateString('%{group} (%{name})', {group: ..., name: ...}, traceObj);
```

So that technically `%{marker.size}` would be equally valid.